test/parallel/test-eslint-prefer-primordials.js

```diff
@@ -137,14 +137,19 @@ new RuleTester({
           const { Number } = primordials;
           Number.parseInt('10')
         `,
-        options: [{ name: 'Number', into: Number }],
+        options: [{ name: 'parseInt', into: 'Number' }, { name: 'Number' }],
```

Comment:
```suggestion
        options: [{ name: 'Number' }],
```

`{ name: 'parseInt', into: 'Number' }` doesn't seem necessary.